Do not submit forms multiple times on CTRL + ENTER

roelvanduijnhoven opened the pull request:
When there are multiple submit buttons, using the CTRL+ENTER shortcut will submit the same form multiple times. This fixes known issue https://github.com/phpmyadmin/phpmyadmin/issues/14460. 

The PMA module adds a lot of duplicate submit buttons. So in that situation it will occur a lot. One example can be seen here, both submit buttons belong to the same form.
![screenshot from 2019-02-22 10-14-22](https://user-images.githubusercontent.com/91910/53232657-d4335a00-368b-11e9-9f1b-93337cdbe468.png)

williamdes reviewed (approved):
LGTM, thank you for the contribution @roelvanduijnhoven !

williamdes commented:
Complicated to reproduce, need more time before merging

roelvanduijnhoven commented:
@williamdes If i can help with something, let me know!

williamdes commented:
@roelvanduijnhoven Can you please send steps so I can reproduce the issue?

roelvanduijnhoven commented:
It occurs when you use the bookmark function that PMA enables. You can see this in action on the demo page:

1. Visit https://demo.phpmyadmin.net/master-config/db_sql.php?db=phpmyadmin
2. Type in a query and press CTRL+ENTER
3. Look in the Console; you see it executed twice.

roelvanduijnhoven commented:
If you need any more help @williamdes: happy to help!

williamdes commented:
@roelvanduijnhoven I reproduced the issue !
Will merge soon :)

Commit:
Fix #14460 -Do not submit forms multiple times on CTRL + ENTER

Fixes: #14460

When there are multiple submit buttons, using the CTRL+ENTER shortcut will submit the same form multiple times.

The PMA module adds a lot of duplicate submit buttons. So in that situation it will occur al ot.

Signed-off-by: Roel van Duijnhoven <roel.duijnhoven@gmail.com>

williamdes reviewed (approved):
Merged, thanks for your contribution @roelvanduijnhoven !

hkunz commented:
I have this same problem! It was about to drive me crazy Ctrl+Enter is executing the statements twice! WTF
